### DISCUSSION TITLE
No validation loss in tensorboard output

### DISCUSSION THREAD

**Discussion by rhingo** [huggingface/autotrain-advanced#713] [General]
Hi,

I am running autotrain through the CLI with a config file. I have separated my dataset into a training and validation split and put that information in the config file as follows:

```
train_split: train
valid_split: test
```
When I try to load the training metrics in tensorboard I only see the training loss vs. epochs. Is there a way I can get the validation loss vs. epochs? 

Thanks!

**Comment by abhishekkrthakur** [comment]
is it llm task?

**Reply by rhingo** [reply]
Yes it is

**Reply by rhingo** [reply]
Just wanted to follow-up on this. Any thoughts on how to get the validation loss vs. epochs?

**Comment by rhingo** [comment]
@abhishekkrthakur Sorry to bother but wanted to check if you have any input on this. Thanks!
